pkg/providers/validator/validate.go

```diff
@@ -37,6 +37,10 @@ func NewIPValidator(opts ...IPValidatorOpt) *IPValidator {
 // ValidateControlPlaneIPUniqueness checks whether or not the control plane endpoint defined
 // in the cluster spec is available.
 func (v *IPValidator) ValidateControlPlaneIPUniqueness(cluster *v1alpha1.Cluster) error {
+	if cluster.SkipControlPlaneIPCheck() {
+		return nil
```

Comment:
it would be interesting to log this

Reply:
I thought of logging this, however this is also used in the CLI and it would need some refactoring to send the log parameter here for logging.